cfe_internal/enterprise/CFE_hub_specific.cf

```diff
@@ -115,19 +115,21 @@ bundle agent update_cli_rest_server_url_config
     "regex_test_pattern" string => ".*localhost:$(cfe_internal_hub_vars.https_port).*";
 
   files:
-   "$(mp_share_config_file)"
-      edit_line => change_cli_rest_server_url_port,
-      if => and(
-        fileexists("$(mp_share_config_file)"),
-        islessthan(countlinesmatching("$(regex_test_pattern)", "$(mp_share_config_file)"), 1)
-        );
-
-    "$(mp_config_file)"
-      edit_line => change_cli_rest_server_url_port,
-      if => and(
-        fileexists("$(mp_config_file)"),
-        islessthan(countlinesmatching("$(regex_test_pattern)", "$(mp_config_file)"), 1)
-        );
+    !mpf_disable_mission_portal_docroot_sync_from_share_gui::
+      "$(mp_share_config_file)"
+         edit_line => change_cli_rest_server_url_port,
+         if => and(
+           fileexists("$(mp_share_config_file)"),
+           islessthan(countlinesmatching("$(regex_test_pattern)", "$(mp_share_config_file)"), 1)
+           );
+
+    any::
+      "$(mp_config_file)"
```

Comment:
This file, since it's being modified after install I think should be marked as a config file for the package manager (rpm/deb) so that it wont trip package verification.

Reply:
let's try this: https://github.com/cfengine/buildscripts/pull/1594

Reply:
I'll let that one work on it's own...